views/home.js

```diff
@@ -51,6 +51,7 @@ export default function HomePageScene({navigator, route}: ScenePropsType) {
     >
       <StatusBar
         barStyle='light-content'
+        backgroundColor={c.gold}
```

Comment:
Nice. Last question: I used `olevilleGold` for the navbar; I think you used `gold` for the status bar? Could you make that consistent, please?